assembly/src/assembler/instruction/mem_ops.rs

```diff
@@ -53,9 +53,14 @@ pub fn mem_read(
 /// this handles mem_store, mem_storew, loc_store, and loc_storew instructions.
 ///
 /// VM cycles per operation:
-/// - mem_store(w): 1 cycle
-/// - mem_store(w).b: 2 cycles
-/// - loc_store(w).b:
+/// - mem_store: 2 cycles
+/// - mem_storew: 1 cyle
+/// - mem_store.b: 3 cycles
+/// - mem_storew.b: 2 cycles
```

Comment:
Should we update these to follow the same format as below (i.e., splitting out `b = 1` vs. `b != 1`)?